servicecontrol/installation.md

```diff
@@ -79,6 +79,20 @@ Clicking the upgrade link will:
 * Run the new binaries to create required queues
 * Start the service
 
+
+To upgrade multiple majors the following approach needs to be taken:
+
+- Upgrade the the latest minor.patch release within a give major
+  - When v1.34.0 is installed, then install v1.48.0 and update all instances
```

Comment:
I would consider pulling the step to update all instances out as a separate section. Maybe above this. 

```
Each instance of ServiceControl needs to be updated separately. Whenever updating to a new version of ServiceControl:

- Download the new installer
- On each machine with ServiceControl instances:
  - Install the new version
  - Open ServiceControl Management and update each instance
```